DDDetectors/src/Beampipe_o1_v01_geo.cpp

```diff
@@ -111,7 +111,7 @@ static dd4hep::Ref_t create_element(dd4hep::Detector& description,
   //Access to the XML File
   xml_det_t xmlBeampipe = element;
   const std::string name = xmlBeampipe.nameStr();
-
+  bool nocore  = xmlBeampipe.nocore(false);
```

Comment:
Ok, I see. I hadn't thought about placing two beampipe detectors. I think then it is OK for me.